src/pages/Course/Course.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect } from "react";
+import { useParams } from "react-router";
+import { useSelector, useDispatch } from "react-redux";
+import { NavLink } from "react-router-dom";
+
+import fetchCourses from "../../services/coursesRequest";
+
+import Button from "../../components/Button/Button";
+
+import "./course.scss";
+
+const Course = () => {
+  const { id } = useParams();
+  const data = useSelector((state) => state.courses);
+  const singleCourse = data.courses.find((props) => props.id === id) || {};
```

Comment:
bad naming
noit is not props inside data.courses.fine, it is course
 const singleCourse = data.courses.find((course) => course.id === id) || {};
 
 but try to get one course by ID from request